Vagrantfile

```diff
@@ -54,12 +83,19 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  # config.vm.synced_folder ".dnf-cache", "/var/cache/dnf", type: "sshfs", sshfs_opts_append: "-o nonempty"
 
  # Comment this line if you would like to disable the automatic update during provisioning
- config.vm.provision "shell", inline: "sudo dnf upgrade -y"
+ #config.vm.provision "shell", inline: "sudo dnf upgrade -y"
```

Comment:
Is this intended for this change? It seems unrelated.

Reply:
yup! an unintended change. turned this off during testing. sorry!

Reply:
No problem, but it seems so be still here...

Reply:
gah, sorry!

i uncommented it, then commented it back out again to speed up testing of the other change, then forgot to uncomment it again! sorry!

just updated the commit again to make sure it is NOT commented out!